scipy/linalg/_basic.py

```diff
@@ -24,6 +26,10 @@
            'pinv', 'pinvh', 'matrix_balance', 'matmul_toeplitz']
 
 
+def arg_err_msg(param):
```

Comment:
will be renamed and imported from `_array_api.py` in a follow up to https://github.com/scipy/scipy/pull/19005.